### DISCUSSION TITLE
as administrator I'd like to create custom navi-menu items instance wide so that users can add-remove from their profile settings

### DISCUSSION THREAD

**Discussion by tassoman** [misskey-dev/misskey#9216] [Ideas]
I wrote this feature as _user story_

> as administrator I'd like to create custom navi-menu items instance wide so that users can add-remove from their profile settings

For example would be intersting having public pages linked inside navigation menu, for example

- Timeline
- Favourites
- Drive
- Explore
- **@sysadmin/pages/tos**

So the navimenu, can print the link to a selected public page.

Users are able to edit their navimenu by editing a textarea.  Maybe we can set a customizable instance wide preset textarea in _Control Panel >> Settings >> Geneneral_
So that at profile creation, users gets the admin's customized menu.

**Comment by ThatOneCalculator** [comment]
Users can already edit the order themselves through Settings > Navigation bar.

If you as an admin want to set defaults, you can edit `packages/client/src/store.ts`.

**Reply by tassoman** [reply]
is it possible to avoid editing the core, having it forked?
